xgcm/grid.py

```diff
@@ -1335,6 +1362,8 @@ def derivative(self, da, axis, **kwargs):
         axis : str
```

Comment:
Could you replace the parameter section with `%(grid_func.parameters)s`. I seem to have forgotten that.